custom/icds_reports/tasks.py

```diff
@@ -1761,3 +1762,20 @@ def drop_df_indices(agg_date):
             cursor.execute(query, params)
         for query in helper.drop_index_queries():
             cursor.execute(query)
+
+
+@periodic_task_on_envs(
+    settings.ICDS_ENVS,
+    run_every=crontab(minute=0, hour=14),  # To run on 7:30 PM IST
```

Comment:
This will always be 1 day behind the aggregation. Is that what we want? Despite our earlier conversation, it may make sense to include this in the agg for data consistency